src/main/java/org/pawaneuler/Generator/TransactionLogGenerator/RandomGenerator.java

```diff
@@ -28,16 +27,17 @@ public RandomGenerator() {
      */
     public RandomGenerator(int maxVariety) {
         this.randomizer = new Random();
-        this.productVariety = randomProducts((maxVariety > 26) ? 26 : maxVariety);
-        this.maxVariety = productVariety.size();
+        this.maxVariety = (maxVariety > 26) ? 26 : maxVariety;
```

Comment:
Explain the reason behind the 26 hard limit of the variety, and if possible better make it into a final variable, like `private final int VARIETY_LIMIT = 26;`.